Formula/minio.rb

```diff
@@ -26,10 +26,10 @@ def install
         system "go", "build", "-o", buildpath/"minio"
       else
         minio_release = `git tag --points-at HEAD`.chomp
-        minio_version = minio_release.gsub(/RELEASE\./, "").chomp
+        minio_version = minio_release.gsub(/RELEASE\./, "").chomp.gsub(/T(\d+)\-(\d+)\-(\d+)Z/, 'T\1:\2:\3Z')
         minio_commit = `git rev-parse HEAD`.chomp
 
-        system "go", "build", "-ldflags", "-X main.minioVersion=#{minio_version} -X main.minioReleaseTag=#{minio_release} -X main.minioCommitID=#{minio_commit}", "-o", buildpath/"minio"
+        system "go", "build", "-ldflags", "-X github.com/minio/minio/cmd.Version=#{minio_version} -X github.com/minio/minio/cmd.ReleaseTag=#{minio_release} -X github.com/minio/minio/cmd.CommitID=#{minio_commit}", "-o", buildpath/"minio"
```

Comment:
Let's split this onto multiple lines since the formula is getting silly wide by this point. @ilovezfs has a nice method to do so; [this sort of thing](https://github.com/Homebrew/homebrew-core/blob/4e8ccbdc0f3e834ed48baaca24f223f8c2f3c503/Formula/doctl.rb#L24-L31).